asv_bench/benchmarks/benchmarks.py

```diff
@@ -384,6 +384,20 @@ def time_drop(self, shape, axis, drop_ncols):
         execute(self.df.drop(self.labels, axis))
 
 
+class TimeHead:
+    param_names = ["shape"]
+    params = [
+        UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
+    ]
+
+    def setup(self, shape):
+        self.df = generate_dataframe(ASV_USE_IMPL, "int", *shape, RAND_LOW, RAND_HIGH)
+        self.head_count = int(0.8 * len(self.df.index))
```

Comment:
it's a kind of unrealistic scenario when we're printing 80% of the frame, can we make fewer amount of rows?

also note, that there is no error if `.head` argument is greater than the number of rows in a frame, so it could be just some fixed value that depends on `ASV_DATASET_SIZE` (maybe [`GROUPBY_NGROUPS`](https://github.com/modin-project/modin/blob/3a3168382fb18f2a41de8858bac0f566415946a9/asv_bench/benchmarks/benchmarks.py#L82))?

Reply:
Added a case with a small number of rows. I don’t want to abandon the case with a large number of lines, because it’s interesting to see the performance in the case when lines from different partitions are needed.

For example, user want to work with half the dataset, in which case, in addition to `iloc/loc` operations, he can use `head` op.